src/Microsoft.ServiceFabric.Actors/Runtime/ActorManager.cs

```diff
@@ -522,7 +522,7 @@ public Task StartLoadingRemindersAsync(CancellationToken cancellationToken)
         /// </summary>
         public async Task<PagedResult<ActorInformation>> GetActorsFromStateProvider(ContinuationToken continuationToken, CancellationToken cancellationToken)
         {
-            // Get the Actors list from State provider and mark them Active or Inactive
+            // Gets the Actors list from State provider and mark them Active or Inactive.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.